app/utils/queryBuilder.ts

```diff
@@ -396,16 +280,29 @@ const buildQuery = (
 
   const where = getWhereExp(allFilterGroups);
 
-  const queryString = BASE_QUERY_STRING.replace(
-    "$queryName",
-    "BuildQuery_" + tableName
-  )
+  /**
+   * If using aggregates, patch in the aggregation query
+   *  - must do this before we patch in the other template
+   * vars
+   */
+  let queryString = BASE_QUERY_STRING.replace(
+    "$aggregationQuery",
+    includeAggregates ? AGGREGATION_QUERY_STRING : ""
+  );
```

Comment:
here's the lil change that enables us to skip the aggregate data query when exporting. the aggregates add a lot of query time and are completely ignored for the purposes of generating the CSV download

Reply:
so clever!!